packages/amplify-graphql-transformer-core/package.json

```diff
@@ -69,7 +69,7 @@
     "coverageThreshold": {
       "global": {
         "branches": 69,
-        "functions": 32,
+        "functions": 31,
```

Comment:
This package hasn't been modified in this PR. Looks this is the current coverage in 'main' branch.